src/flb_engine.c

```diff
@@ -458,6 +459,7 @@ int flb_engine_start(struct flb_config *config)
 
     /* Initialize the networking layer */
     flb_net_init();
+    flb_pack_set_null_as_nan(config->convert_nan_to_null);
```

Comment:
can you add a flb_pack_init() into flb_pack.c that take care of this config ?

Reply:
Thank you for reviewing.
I updated.